### ISSUE TITLE
Experienced training crash after switching from 8 GPUs to 16 GPUs

### ISSUE THREAD

**Issue by guoran-future-hu** [flymin/MagicDriveDiT#60]
Hello! We were trying to train stage 3 with some architecture modification. We started training with 8 GPUs due to resource limitation, then attempted to switch to 16 GPUs by continuing from the checkpoint. However, the loss became nan after 3k training steps and inferencing the model only produces black screen.

Is changing the number of GPUs mid-training supported in your framework? If so, are there any special steps we need to take when making this transition? Or is it best to use the same GPU configuration throughout the training?

Thanks for your help!

**Comment by flymin** [comment]
This could happen in any stage, but we do not know the exact cause. Please try to use warmup or resume from other ckpts.
